### ISSUE TITLE
Unhandled exception (#7882d899)

### ISSUE THREAD

**Issue by sqlmapreporter** [sqlmapproject/sqlmap#5805]
```
Running version: 1.8.4.5#dev
Python version: 3.8.8
Operating system: Windows-10-10.0.19041-SP0
Command line: sqlmap.py
Technique: None
Back-end DBMS: None
```
```
Traceback (most recent call last):
  File "sqlmap.py", line 222, in main
    start()
  File "lib/core/decorators.py", line 80, in _
    result = f(*args, **kwargs)
  File "lib/controller/controller.py", line 300, in start
    for targetUrl, targetMethod, targetData, targetCookie, targetHeaders in kb.targets:
  File "lib/core/option.py", line 433, in __next__
    return self.next()
  File "lib/core/option.py", line 437, in next
    line = next(conf.stdinPipe)
  File "C:/Users/*************/anaconda3/lib/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
```

**Comment by projectlocalhost** [comment]
> ```
> Running version: 1.8.4.5#dev
> Python version: 3.8.8
> Operating system: Windows-10-10.0.19041-SP0
> Command line: sqlmap.py
> Technique: None
> Back-end DBMS: None
> ```
> 
> ```
> Traceback (most recent call last):
>   File "sqlmap.py", line 222, in main
>     start()
>   File "lib/core/decorators.py", line 80, in _
>     result = f(*args, **kwargs)
>   File "lib/controller/controller.py", line 300, in start
>     for targetUrl, targetMethod, targetData, targetCookie, targetHeaders in kb.targets:
>   File "lib/core/option.py", line 433, in __next__
>     return self.next()
>   File "lib/core/option.py", line 437, in next
>     line = next(conf.stdinPipe)
>   File "C:/Users/*************/anaconda3/lib/codecs.py", line 322, in decode
>     (result, consumed) = self._buffer_decode(data, self.errors, final)
> UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
> ```

**Comment by stamparm** [comment]
@projectlocalhost that's fixed in 1.8.11.4
